docs/subnets/create-a-virtual-machine-vm.md

```diff
@@ -1,294 +1,22 @@
----
-sidebar_position: 4
----
+# How to Build a Simple VM
 
-# Create a Virtual Machine (VM)
+This is part of a series of tutorials for building a Virtual Machine (VM):
 
-## Introduction
+- [Introduction to Virtual Machines](./introduction-to-vm.md)
+- How to Build a Simple VM (this article)
+- [How to Build a Complex VM](./create-a-vm-blobvm.md)
 
-One of the core features of Avalanche is the ability to create new, custom blockchains, which are defined by [Virtual Machines (VMs)](../overview/getting-started/avalanche-platform.md#virtual-machines)
+## Introduction
 
-In this tutorial, we’ll create a very simple VM. The blockchain defined by the VM is a [TimestampVM](https://github.com/ava-labs/timestampvm/tree/v1.2.1). Each block in the blockchain contains the timestamp when it was created along with a 32-byte piece of data (payload). Each block’s timestamp is after its parent’s timestamp.
+In this tutorial, we’ll create a very simple VM. The blockchain defined by the VM is the [TimestampVM](https://github.com/ava-labs/timestampvm/tree/v1.2.1). Each block in the TimestampVM's blockchain contains the timestamp when it was created along with a 32-byte piece of data. Each block’s timestamp is after its parent’s timestamp.
```

Comment:
This argument below about why the timestampvm is useful is a pretty tough sell. I think the purpose of this VM was as an example we don't really need to justify it anymore than that. Can we remove the paragraph below about how it will be useful for storing a book manuscript?